onnx2webnn/onnx2webnn.py

```diff
@@ -348,6 +357,46 @@ def transpose_external_weights(filter_name, permutation):
     );""")
             return filter_var_name
 
+        # Get shape info for a tensor by name
+        def get_tensor_shape(model, tensor_name):
```

Comment:
I introduced a similar helper for QDQ support: https://github.com/huningxin/onnx2webnn/blob/main/onnx2webnn/onnx2webnn.py#L418. Please take a look and rebase if it is good.

Reply:
It looks good, I will rebase it.